meetings/membership-criteria.md

```diff
@@ -0,0 +1,64 @@
+# Membership criteria
+
+When you apply for membership, we review your application based on the [membership participation types](membership-participation-types.md) defined. We ask for some basic information about you. 
+
+## Basic information
+
+First, we validate the personal information:
+
+- Reasonable first name, last name.
```

Comment:
Reasonable might be questioned. Instead

First Name and Last Name are reviewed to eliminate potential 'bots'.